py/aou_workbench_client/concepts.py

```diff
@@ -62,6 +64,129 @@
     ('Vocabulary', 'vocabulary_id'),
     ('Count', 'count_value')]
 
+_CONCEPT_TABLE_HTML_TEMPLATE = """
+<script language="javascript">
+  var selected_row_id = null;
```

Comment:
Could you use es6 instead? s/var/let/

It may complain that you need to be in a function. This may be a good idea anyways - I'm not sure how much this script is sandboxed from the rest of the page by ipynb. If it's not, then declaring variables like this might conflict with other global vars.

You can scope your Javascript by wrapping the whole thing in an anonymous function which is immediately invoked:

```
() => {
  ... code ...
}();
```

Reply:
This will be addressed by RW-796 (if we keep this code around as a widget.)